src/fides/api/ops/schemas/saas/saas_config.py

```diff
@@ -355,6 +355,7 @@ class SaaSConfig(SaaSConfigBase):
 
     description: str
     version: str
+    replaceable: bool = False
```

Comment:
I decided to flag this in the SaaS config to avoid a metadata file in the zip file. I thought it'd be fine since we can default this to false for a majority of the uses cases.